samples/counter/ios-shared/build.gradle

```
@@ -15,17 +15,15 @@
  */
 
 apply plugin: 'org.jetbrains.kotlin.multiplatform'
-apply plugin: 'org.jetbrains.kotlin.native.cocoapods'
 apply plugin: 'app.cash.redwood'
 
 kotlin {
-  iosArm64()
-  iosX64()
-  iosSimulatorArm64()
-
-  cocoapods {
-    noPodspec()
-    framework {
+  [
+    iosArm64(),
+    iosX64(),
+    iosSimulatorArm64(),
+  ].each { iosTarget ->
```

I _think_ we can just do this:
```suggestion
  targets.each { iosTarget ->
```

---

Moreover, it's super weird that we can invoke these functions twice and still only create a single target.

So, alternatively, consider deleting the three lines above and only having
```kotlin
[ iosArm64(), etc. ].each { .. }
```
so that they're only invoked once.

---

> Moreover, it's super weird that we can invoke these functions twice and still only create a single target.

The first invocation of the function is happening within the list constructor. Where's the invocation of the function happening the second time?

---

Oh you deleted the original ones. Sorry reviewing on my phone so harder to keep track of what's going on.